src/views/shared/Footer.js

```diff
@@ -72,6 +76,12 @@ function (Okta) {
     },
     toggleLinks: function (e) {
       e.preventDefault();
+      var revertedVal = !(
```

Comment:
might be easier if we use the visibility of the element... `this.$('.js-help').is(':visible')`

Reply:
Because it has an animation we need to wait in order to use this approach.